libfwupdplugin/fu-context.h

```diff
@@ -85,3 +86,10 @@ guint
 fu_context_get_battery_threshold(FuContext *self);
 void
 fu_context_set_battery_threshold(FuContext *self, guint battery_threshold);
+
+FuBiosAttrs *
+fu_context_get_bios_attrs(FuContext *self);
+gboolean
+fu_context_get_bios_attr_pending(FuContext *self);
+FwupdBiosAttr *
+fu_context_get_bios_attr(FuContext *self, const gchar *name);
```

Comment:
I wonder if we want s/fu_context_get_bios_attr/fu_context_get_bios_attr_by_name as I can imagine we want the same thing by *path* in the future.

Reply:
Actually by path is very unlikely, usually one attribute provider is available for a machine.  If we end up with two, I think we'll have other things to look at to avoid potential collisions .
IOW cross this bridge later IFF it comes up.